styles/style.css

```diff
@@ -0,0 +1,51 @@
+* {
+    box-sizing: border-box;
+}
+body {
+    font-family: 'Roboto', sans-serif;
+    color: #383838;
+    margin: 0;
+}
+.offer-container {
```

Comment:
please add `margin: 0 auto;`
this will put your container in the center

Reply:
Done